### ISSUE TITLE
is it possible to replace the whisper model with the finetune one

### ISSUE THREAD

**Issue by Tomtunn** [m-bain/whisperX#1237]
I’d like to run WhisperX using my own fine-tuned Whisper model instead of the default OpenAI weights. is its possible, if possible HOW ?

### What I want to load
- Model format: <Hugging Face Transformers / CTranslate2 (faster-whisper) / original OpenAI .pt>
- Location: <HF repo ID or local path> (e.g., `username/whisper-finetune-xyz`)

### What I tried
```python
import whisperx
model = whisperx.load_model("<repo form hugging face>", device="cuda", compute_type="float16")

**Comment by lslot-DL** [comment]
Yes, just provide the local path to your finetuned model. However, please keep in mind whisperX requires you to turn your model into a CTranslate one. Here's code that I use for the conversion:

```python
from transformers import WhisperForConditionalGeneration, WhisperProcessor
import ctranslate2
import os
import shutil

# Load your fine-tuned model
model = WhisperForConditionalGeneration.from_pretrained(input_model_path)
processor = WhisperProcessor.from_pretrained(input_model_path)

# Create output directory
os.makedirs(output_model_path, exist_ok=True)

# Convert using ctranslate2 with local path
converter = ctranslate2.converters.TransformersConverter(
    model_name_or_path=input_model_path,
    low_cpu_mem_usage=True,
)

converter.convert(
    output_dir=output_model_path,
    quantization="float16",
    force=True
)

# Copy tokenizer files to output directory
tokenizer_files = ["tokenizer.json", "vocab.json", "merges.txt", "normalizer.json", 
                    "tokenizer_config.json", "special_tokens_map.json"]

for file_name in tokenizer_files:
    src_path = os.path.join(input_model_path, file_name)
    dst_path = os.path.join(output_model_path, file_name)
    if os.path.exists(src_path):
        shutil.copy2(src_path, dst_path)
    
    # Copy tokenizer files to output directory
    tokenizer_files = ["tokenizer.json", "vocab.json", "merges.txt", "normalizer.json", 
                      "tokenizer_config.json", "special_tokens_map.json"]
    
    for file_name in tokenizer_files:
        src_path = os.path.join(input_model_path, file_name)
        dst_path = os.path.join(output_model_path, file_name)
        if os.path.exists(src_path):
            shutil.copy2(src_path, dst_path)
```
